eventdata/challenges/bulk-update.json

```diff
@@ -21,32 +19,6 @@
     {
       "operation": "create-index-template"
     },
-    {
-      "operation": {
-        "operation-type": "create-index",
-        "index": "elasticlogs",
-        "body": {
-          "settings": {
```

Comment:
All of these settings can be defined via track parameters on the index template.